packager/certcache/certcache.go

```diff
@@ -240,7 +264,8 @@ func (this *CertCache) readOCSP() ([]byte, time.Time, error) {
 	if len(ocsp) == 0 {
 		return nil, time.Time{}, errors.New("Missing OCSP response.")
 	}
-	if !this.isHealthy(ocsp) {
+	health, err := this.isHealthy(ocsp)
+	if err != nil || !health {
 		return nil, time.Time{}, errors.New("OCSP failed health check.")
```

Comment:
Use `errors.Wrap()` instead of `errors.New()`. This way, the detailed message & stack trace will eventually bubble up to a log statement.

Reply:
Done.